packages/backend-openapi-utils/src/types/common.ts

```diff
@@ -220,12 +220,11 @@ export type TuplifyUnion<
 /**
  * @public
  */
-export type ConvertAll<T, R extends ReadonlyArray<unknown> = []> = T extends [
-  infer First extends JSONSchema7,
-  ...infer Rest,
-]
-  ? ConvertAll<Rest, [...R, FromSchema<First>]>
-  : R;
+export type ConvertAll<T extends ReadonlyArray<unknown>> = {
```

Comment:
This change is because newer versions of TS complained about excessive/infinite complexity for the old implementation.

@sennyeya does this match your expectations? I believe it should achieve roughly the same, except that I think the old implementation actually got short circuited whenever it encountered a single non-schema in the tuple, while the new one continues going.